.github/workflows/update-website.yml

```
@@ -0,0 +1,50 @@
+name: Update latest generator documentation in the website
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'docs/*.md'
```

```suggestion
      - 'docs/*.md'
      - '!docs/README.md'
```

taken from https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-paths